Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,83 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local IFS=$'\n'
+  local basedir="${1/\~/$HOME}"
+  _values -C "packages from $basedir" ${$( find -L "$basedir" -mindepth 1 -maxdepth 1 \( -name .git -o -name .svn \) -prune -o -type d -print 2>/dev/null | sed "s#^${basedir}/##" | sort -u ):-""}
+}
+
+_stow(){
+  local context state line curcontext="$curcontext" ret=1
+  typeset -A opt_args
+  # Others local variables
+  local stow_dir arguments
+
+  arguments=(
+    '(- *)'{--help,-h}'[show help]'
+    '(- *)'{--version,-v}'[show version number]'
+    '(-d --dir)'{-d,--dir=-}'[set the stow dir (default is current dir)]:stow dir:_files -/'
+    '(-t --target)'{-t,--target=-}'[set the target dir (default is parent of stow dir)]:target dir:_files -/'
+    '*'{-S,--stow}'[stow the package names that follow]:stow package:->stow_package'
+    '*'{-D,--delete}'[unstow the package names that follow]:stow package:->stow_package'
+    '*'{-R,--restow}'[restow (unstow and stow again) the package names that follow]:stow package:->stow_package'
+    '--adopt[adopt already existing plain file]'
+    '--ignore=-[ignore files ending with this perl regex]:regexp:'
+    "--defer=-[don't stow files beginning with this perl regex]:regexp:"
+    '--override=-[force stowing files beginning with this perl regex]:regexp:'
+    '--no-folding[disable any further tree folding or tree refolding]'
+    '(-p --compat)'{-p,--compat}'[use legacy algorithm for unstowing]'
+    '(-n -no --simulate)'{-n,--no,--simulate}'[do not actually make any filesystem changes]'
+    '(-v --verbose)'{-v,--verbose}'[increase verbosity]'
+    '(-v --verbose)'--verbose=-'[set verbose level to n]:level:(o 1 2 3 4 5)'
+    '*:stow package:->stow_package'
+  )
+
+  if (( ${+commands[stow]} )) ; then
+    if [[ "$( _call_program version stow --version 2>/dev/null )" =~ "stow\ \(GNU\ Stow\)\ version\ ([0-9]+\.[0-9]+\.[0-9]+)" ]]; then
+      autoload -z is-at-least
```

Comment:
`autoload -Uz` ?

Reply:
indeed, according to man zshmisc _This is recommended for the use of functions supplied with the zsh distribution_ :disappointed_relieved: